art/tests/test.py

```diff
@@ -4682,6 +4682,22 @@
 /  (- _)  /
 <BLANKLINE>
 <BLANKLINE>
+>>> tprint("abcd", mode=PrintingMode.LINE)
```

Comment:
It seems testing this feature is a bit tricky, so please remove it from this PR. We will add it later.